distributed/utils_test.py

```diff
@@ -856,9 +856,9 @@ async def start_cluster(
 async def end_cluster(s, workers):
     logger.debug("Closing out test cluster")
 
-    async def end_worker(w):
+    async def end_worker(w: Worker):
         with suppress(TimeoutError, CommClosedError, EnvironmentError):
-            await w.close(report=False)
+            await w.close(report=False, timeout=2)
```

Comment:
2 seconds seems relatively small compared to other timeouts I see in this module. Should we be concerned about workers needing more than 2 seconds to shut down in non-pathological cases?

Reply:
Yeah, that probably is too short. Happy to change to something longer, though I think if you're trying to close the cluster because the test is over (either due to completion or error), I'd guess that in almost all cases it's either fractions of a second until threads end successfully, or they will never end successfully (or they're blocked on something with its own timeout, and who knows how long that is).

In the world of slow CI, maybe we make this 10 seconds?